docs/en/04_Changelogs/4.10.0.md

```diff
@@ -1,19 +1,119 @@
-# 4.10.0 (unreleased)
+# 4.10.0
 
 ## Overview
 
+A full list of module versions included in CMS Recipe 4.9.0 is provided below. We recommend referencing recipes in your dependencies, rather than individual modules, to simplify version tracking. See [Recipes](/getting_started/).
```

Comment:
```suggestion
A full list of module versions included in CMS Recipe 4.10.0 is provided below. We recommend referencing recipes in your dependencies, rather than individual modules, to simplify version tracking. See [Recipes](/getting_started/).
```

Reply:
Fixed